controllers/managedocs_controller_test.go

```diff
@@ -373,13 +402,37 @@ var _ = Describe("ManagedOCS controller", func() {
 				By("Creating a storagecluster resource")
 				utils.WaitForResource(k8sClient, ctx, scTemplate.DeepCopy(), timeout, interval)
 
-				By("Creating a prometous resource")
+				By("Creating a prometheus resource")
 				utils.WaitForResource(k8sClient, ctx, promTemplate.DeepCopy(), timeout, interval)
 
 				By("Creating an alertmanager resource")
 				utils.WaitForResource(k8sClient, ctx, amTemplate.DeepCopy(), timeout, interval)
 			})
 		})
+		When("there is no rook-ceph-operator-config ConfigMap", func() {
+			It("should not create reconciled resources", func() {
+				resList := []runtime.Object{
+					scTemplate.DeepCopy(),
+					promTemplate.DeepCopy(),
+					amTemplate.DeepCopy(),
+				}
+
+				// Delete existing resources
+				for _, object := range resList {
+					Expect(k8sClient.Delete(ctx, object)).Should(Succeed())
+				}
+
+				// Ensure there is no rook-ceph-operator-config ConfigMap present
+				configMap := rookConfigMapTemplate.DeepCopy()
+				Expect(k8sClient.Delete(ctx, configMap)).Should(Succeed())
+
+				// Ensure, over a period of time, that the resources are not created
+				utils.EnsureNoResources(k8sClient, ctx, resList, timeout, interval)
+
+				// Recreate the configMap for other tests.
+				Expect(k8sClient.Create(ctx, configMap)).Should(Succeed())
```

Comment:
If you are deleting resources in the test you need to verify and wait for them to be create (by the operator) in the end of the test